src/tests/ftest/datamover/copy_procs.py

```diff
@@ -45,8 +45,8 @@ def test_copy_procs(self):
         """
         # Create pool and containers
         pool1 = self.create_pool()
-        cont1 = self.create_cont(pool1)
-        cont2 = self.create_cont(pool1)
+        cont1 = self.add_container(pool1)
+        cont2 = self.add_container(pool1)
```

Comment:
This is the paradigm I'd ultimately like to see. `add_container` adds the container to the internal list, but the test case doesn't access `self.container` directly